src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -70,19 +75,17 @@ export const applicantAddressInfoSchema = {
     applicantNewAddress: {
       ...radioUI({
         updateUiSchema: formData => {
+          const name = nameWording(formData, undefined, false, true);
           const labels = {
             yes: 'Yes',
             no: 'No',
             unknown: 'I’m not sure',
           };
 
           return {
-            'ui:title': `Has ${nameWording(
-              formData,
-              undefined,
-              undefined,
-              true,
-            )} mailing address changed since their last CHAMPVA form submission?`,
+            'ui:title': `Has ${name} mailing address changed since ${
+              name === 'your' ? name : 'their'
+            } last CHAMPVA form submission?`,
```

Comment:
Updated the custom wording so it's more comprehensive. This same type of change was done in several places.